RELEASE_NOTES.md

```diff
@@ -3,6 +3,7 @@
 * Improved logging
 * Changed CurrentModel and UpdateModel on ViewModel<_,_> from public to internal
 * Added support for composable binding validation
+* Chagned minimum Elmish version from 3.0.3 to 3.1.0 (which is currently the latest)
```

Comment:
Typo. Also, this point should perhaps mention something to the effect that juggling around synchronization context for the relevant dialogs are no longer needed.

Reply:
Just force pushed a rebase onto `master` that also fixes that typo and mentions the execution happening on the threadpool.

Reply:
Thanks. That is useful. I'm thinking however that we should also explicitly mention that code related to save/load dialogs can be simplified. Otherwise people will have no idea they can simplify this unless they diff the readme and see that the FAQ entry is removed. We don't have to provide a lot of detail though; we can link to the `Program.fs` diff in the FileDialogs sample. Something like this:

> This means that e.g. Save File / Open File dialogs can be simplified. See [this diff for details](https://github.com/elmish/Elmish.WPF/pull/349/files#diff-48689f4b515bf41b1bdad0aee209185e63ee109980cff7c8a4bd2996bc318aa2).

But replace the link with the actual commit diff after merging. (Not idea if this can be done before merging.)

Reply:
Sure.  This can be done by splitting the commits, ordering the commits, pushing the first two, and then including the link to the second commit diff in the third commit.  I just force pushed those commits.

Reply:
Yes, but now the link points to the commit in the context of the PR. After merging, I suggest updating the link to a "raw" diff view of the commit, to reduce noise. As far as this PR goes, though, this is fine.

Reply:
Oh, ok.  I didn't think carefully about that link.  It contained the correct git commit hash but not the right GitHub URL.  I replaced it with the correct URL, which won't work until I complete this PR...because the branch in this PR is in my fork instead of in this repo.

Reply:
I merged the PR.  [This is the link in the release notes](https://github.com/elmish/Elmish.WPF/commit/d1ec823ccd7f377a860b76bc2358706dc6a70c84), which is working now.